File: C++/Product.cpp
#include <bits/stdc++.h>

using namespace std;

class Product
// kelas yang digunakan untuk mengimplementasikan product
{
private:
    // private atribute dari kelas product
    int price;
    int idProduct;

public:
    // konstrutor
    Product() {}

    // mengeset nilai atribut price
    void setPrice(int price)
    {
        this->price = price;
    }

    // mengembalikan nilai atribut price
    int getPrice()
    {
        return this->price;
    }

    // mengeset nilai atribut idProduct
    void setIdProduct(int idProduct)
    {
        this->idProduct = idProduct;
    }

    // mengembalikan nilai atribut idProduct
    int getIdProduct()
    {
        return this->idProduct;
    }

    // menampilkan atribut product
    void printProduct()
    {
        cout << "Id Product : " << this->getIdProduct() << endl;
        cout << "Price : " << this->getPrice() << endl;
    }

    // destruktor
    ~Product() {}
};

File: Python/Product.py
class Product:
    # kelas yang digunakan untuk mengimplementasikan kelas Product
    def __init__(self):
        # konstruktor
        self.price = 0
        self.idProduct = 0

    def setPrice(self, price):
        # mengeset nilai atribut price
        self.price = price

    def getPrice(self):
        # mengembalikan nilai atribut price
        return self.price

    def setIdProduct(self, idProduct):
        # mengeset nilai atribut idProduct
        self.idProduct = idProduct

    def getIdProduct(self):
        # mengembalikan nilai atribut idProduct
        return self.idProduct

    def printProduct(self):
        # menampilkan atribut product
        print("Id Product: " + str(self.getIdProduct()))
        print("Price: " + str(self.getPrice()))
